framework/src/outputs/Checkpoint.C

```diff
@@ -47,13 +47,18 @@ Checkpoint::validParams()
 
   // Advanced settings
   params.addParam<bool>("binary", true, "Toggle the output of binary files");
+  params.addParam<int>(
+      "autosave_time_interval",
+      600,
+      "Time in seconds that the checkpoint should wait to print out an automatic backup.");
   params.addParamNamesToGroup("binary", "Advanced");
   return params;
 }
 
 Checkpoint::Checkpoint(const InputParameters & parameters)
   : FileOutput(parameters),
     _is_autosave(getParam<AutosaveType>("is_autosave")),
+    _autosave_time_interval(std::chrono::seconds(getParam<int>("autosave_time_interval"))),
```

Comment:
```suggestion
    _autosave_time_interval(std::chrono::seconds(getParam<unsigned int>("autosave_time_interval"))),
```

Reply:
Looks like the orders match to me.  Wouldn't *we* be getting a compiler error for that?  `-Wreorder` in gcc is included in `-Wall`, and I thought basically all our recipes had `ADDITIONAL_CPPFLAGS = -Wall -Wextra -Werror` set.

Reply:
Yeah the order is correct, they're the first and second variables in the header respectively.